Makefile

```diff
@@ -1,15 +1,59 @@
-.PHONY: serve html clean github
+# type `make help` to see all options 
 
-serve:
-	@hugo --i18n-warnings server -D
+# If you have naively forked this repo, say to
+# github.com://myname/numpy.org.git, you can test out the build via
+# make TARGET=myname BASEURL=https://myname.github.io/numpy.org deploy
 
-html:
-	@hugo
-	@touch public/.nojekyll
+TARGET ?= origin
+BASEURL ?= 
 
-clean:
-	@rm -rf public
+ifeq ($(TARGET), origin)
+	WORKTREETARGET =
```

Comment:
This does not work, it checks out `gh-pages` on the same commit as master for me. Why is the `if/else` needed at all? `$(TARGET)/gh-pages` has the correct default already.

Reply:
removed.